include/odp/api/spec/ipsec.h

```diff
@@ -1206,6 +1210,12 @@ typedef struct odp_ipsec_status_t {
  * restored. The amount and content of packet data before the IP header is
  * undefined.
  *
+ * Additional TFC padding might be present after packet contents for ESP tunnel
+ * mode. Received side can use total (IPv4) or payload (IPv6) length from
```

Comment:
The term "TFC padding" in this API proposal seems to differ from that in the RFC. It appears that here TFC padding refers only to padding after IP payload in tunnel mode.

TFC padding, as defined in the RFC, is not restricted to tunnel mode. Transport mode which carries e.g. UDP or IP or other suitable protocol can have TFC padding but that can not be recognized by ODP in the general case (e.g. with proprietary IP protocols).

Reply:
... my point being, the description should be improved to make it more accurate and unambiguous.

Reply:
I will update wording to match RFC. Noted that ODP does not touch TFC padding in transport mode.